test/vast_parser.js

```diff
@@ -137,7 +137,7 @@ describe('VASTParser', function() {
     });
 
     it('should have returned a VAST response object', () => {
-      this.response.should.be.an.instanceOf(VASTResponse);
+      isVASTResponse(this.response).should.eql(true);
```

Comment:
I don't think it is a good idea to introduce a function in the production code just for unit tests. You should instead make sure that `this.response` has the correct properties with the correct values explicitly in the unit test.